site/en/blog/keyboard-focusable-scrollers/index.md

```diff
@@ -0,0 +1,192 @@
+
+In such cases, many web users use the up down motions from their mouse or touchpad to scroll across the element. However, a pointing device, trackpad, or touchscreen is not  every user's optimal way to navigate a page. Some people prefer to navigate across an HTML page accessing every focusable element using only their keyboard. This can be for a variety of reasons. From people who have tremors or other issues that make it difficult to operate a mouse, those who have difficulty visually locating the mouse cursor, and others who use a single switch or voice control.
+
+Accessibility best practices recommend that all features must be available via keyboard access. This way, everyone can use the web in the way that works best for them.
+
+> [2.1.1](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#keyboard-operation-keyboard-operable) Keyboard: All [functionality](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html#functiondef) of the content is operable through a [keyboard interface](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html#keybrd-interfacedef) without requiring specific timings for individual keystrokes, except where the underlying function requires input that depends on the path of the user's movement and not just the endpoints. (Level A)
+
+{% Aside %}  
+Learn more about [keyboard focus](https://web.dev/learn/accessibility/focus) in our Learn Accessibility course.  
+{% endAside %}
+
+## Before this change to focus in scrollers
+
+Prior to this change, a scroller element can only be tab focused if the tabindex is explicitly set to 0 or higher. For example, use the following CSS and HTML. Then, try to `tab` from the first button to the scroller element.
+
+```css
+div.scroll, button {
+  border: 1px solid lightgray;
+  margin-top: 1em;
+  border-radius: 0.5em;
+}
+
+div.scroll {
+  overflow: auto;
+  width: 20em;
+  height: 5em;
+  display: block;
+}
+div.long {
+  width: 10em;
+  height: 10em;
+}
+```
+
+```html
+<button>Start</button>
+<div class="scroll" tabindex="0">
+<p>This is a tab focusable scroller...</p>
+<div class="long"></div>
+<p>You need to scroll down to see this line.</p>
+</div>
+<button>End</button>
+```
+
+<figure>
+{% Video
+  src="video/kheDArv5csY6rvQUJDbWRscckLr1/tHnEyxfzYBbx9D8I1LuB.mp4",
+  autoplay="true",
+  loop="true",
+  muted="true",
+  controls="true"
+%}
+    <figcaption>The scroller is focusable due to a positive tabindex.</figcaption>
+</figure>
+
+Using a negative tabindex as in the following HTML, the scroller will be skipped.
+
+```html
+<button>Start</button>
+<div class="scroll" tabindex="-1">
+<p>This is not a tab focusable scroller...</p>
+<div class="long"></div>
+<p>You cannot see this line using the keyboard.</p>
+</div>
+<button>End</button>
+```
+
+<figure>
+{% Video
+  src="video/kheDArv5csY6rvQUJDbWRscckLr1/NCYssAUl1Ne9wl2A7tr4.mp4",
```

Comment:
Nit: Would be nice if the videos are a smaller so it looks less pixelized on a default screen.